cpp/modmesh/buffer/pymod/TypeBroadcast.hpp

```diff
@@ -234,3 +234,5 @@ struct TypeBroadcast
 
 } /* end namespace python */
 } /* end namespace modmesh */
+\r\n
```

Comment:
It seems that you have some lines mixed with [dos/windows CF+LF plain-text line ending](https://www.cs.toronto.edu/~krueger/csc209h/tut/line-endings.html).  Please correct them.

You probably did not configure your editor correctly.